api/src/main/resources/cb_review_api.yaml

```diff
@@ -692,6 +692,7 @@ definitions:
         description: The Value of the annotation if the AnnotationType is DATE
       annotationValueBoolean:
         type: boolean
+        default: null
```

Comment:
@freemabd When I defaulted this to false, it broke a test case [here](https://github.com/all-of-us/workbench/blob/d2e1064dfbb793036a1e146613abbf00478122b8/api/src/test/java/org/pmiops/workbench/api/CohortReviewControllerTest.java#L599). I didn't quite follow, but the test assumed something not to be set that was getting set when I made `annotationValueBoolean` default to false.

Reply:
null is the preferred default in this case, because this change will cause `annotationValueBoolean` to persist as false instead of null if it isn't provided by the client.

Reply:
Could we change it to `annotationValueNullableBoolean`?

Let me think how to write a unit test for those cases.

Reply:
We can if needed.

Reply:
OK. I'll defer to you if you think that's desirable. Ideally you'd change the `AnnotationType` to `NULLABLE_BOOOLEAN` at the same time if possible.